packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -134,6 +141,12 @@ export class FluidDevtools implements IFluidDevtools {
 	 */
 	private readonly containers: Map<ContainerKey, ContainerDevtools>;
 
+	/**
+	 * Stores Container Runtime-level devtools instances registered with this object.
+	 * Maps from a {@link ContainerKey} to the corresponding {@link ContainerRuntimeDevtools} instance.
+	 */
+	private readonly containerRuntimes: Map<ContainerKey, ContainerRuntimeDevtools>;
```

Comment:
I think you can probably combine the 2 maps you have here. Just:

```typescript
	private readonly containers: Map<ContainerKey, BaseDevtools>;
```

Reply:
I kept separate lists for the convenience of implementing the separate view pane in the devtools-view [comment](https://github.com/microsoft/FluidFramework/pull/25137#discussion_r2261455870). If we are set to unifying the pane to just `Container`, I will also make the change here accordingly.